src/ga4gh/vrs/models.py

```diff
@@ -188,7 +188,7 @@ def _recurse_ga4gh_serialize(obj):
         return obj
 
 
-class _ValueObject(DomainEntity, ABC):
+class _ValueObject(Entity, ABC):
```

Comment:
I think this is fine, but to be consistent with VRS I wonder if we should remove `_ValueObject` and just extend Entity?